packages/valory/skills/apy_estimation_abci/models.py

```diff
@@ -53,12 +54,17 @@ class FantomSubgraph(ApiSpecs):
     """A model that wraps ApiSpecs for Fantom subgraph specifications."""
 
 
-class SpookySwapSubgraph(ApiSpecs):
-    """A model that wraps ApiSpecs for SpookySwap subgraph specifications."""
+class ETHSubgraph(ApiSpecs):
+    """A model that wraps ApiSpecs for ETH subgraph specifications."""
```

Comment:
What is this class used for?

Reply:
It will be later used in #1105 for fetching the blocks from ethereum, which is necessary for Uniswap.